motionplan/tpspace/ptgGroupFrame.go

```diff
@@ -25,18 +25,21 @@ const (
 
 // If refDist is not explicitly set, default to pi radians times this adjustment value.
 const (
-	refDistHalfCircles = 0.9
-	defaultTrajCount   = 3
+	defaultRefDistFar         = 100000 // 100 meters
+	defaultRefDistHalfCircles = 0.9
+	defaultTrajCount          = 2
 )
 
 type ptgFactory func(float64, float64) PTG
 
-var defaultPTGs = []ptgFactory{
+var defaultRestrictedPtgs = []ptgFactory{
 	NewCirclePTG,
 	NewCCPTG,
+}
+
+var defaultPTGs = []ptgFactory{
 	NewCCSPTG,
 	NewCSPTG,
-	NewSideSPTG,
```

Comment:
This PTG is unnecessary with the other updates that have been made, and more PTGs = more runtime.